Azure/azure-ad/ingest/parser.yml

```diff
@@ -0,0 +1,83 @@
+name: azure-ad
+pipeline:
+  - name: parsed_event
+    external:
+      name: json.parse-json
+  - name: azuread
+  - name: azuread-filters
+  - name: set_action_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+  - name: set_if_message_properties
+    filter: '{{ parsed_event.message.properties != null }}'
+
+stages:
+  azuread:
+    actions:
+      - set:
+          service.type: "ldap"
+          service.name: "active directory"
+          log.hostname: "azureactivedirectory"
+          action.name: '{{parsed_event.message.operationName}}'
+          user.id: '{{ parsed_event.message.userId}}'
+          user.name: '{{ parsed_event.message.userPrincipalName}}'
+          action.outcome: '{{parsed_event.message.properties.result}}'
+          source.ip: '{{parsed_event.message.ipAddress}}'
+          source.address: '{{parsed_event.message.ipAddress}}'
```

Comment:
Not required as The ingest worker set `source.address` from `source.ip` or `source.domain`
```suggestion
```